buildroot-external/rootfs-overlay/usr/sbin/hassos-supervisor

```diff
@@ -11,6 +11,7 @@ set -e
 # Init supervisor
 SUPERVISOR_DATA=/mnt/data/supervisor
 SUPERVISOR_STARTUP_MARKER="/run/supervisor/startup-marker"
+SUPERVISOR_STARTSCRIPT_VERSION="${SUPERVISOR_DATA}/.hassos-supervisor-version"
```

Comment:
I think we can but it to `/mnt/data/` or `/var/lib/supervisor/`, I don't think the supervisor need or should have access to this file. But that is my only comment on it, others looks fine I guess.

Reply:
Hm, `/var/lib/supervisor/` would need another bind mount. Its not that important, we can move it easily if needed. I'll just store it in `/mnt/data/` for now.